server/budgettracker.web/Controllers/BudgetApiController.cs

```diff
@@ -35,5 +35,18 @@ public async Task<IActionResult> CreateBudget(ApiRequest request)
                 return Forbid();
             }            
         }
+
+        [HttpPost("update")]
+        public async Task<IActionResult> UpdateBudget(ApiRequest request)
```

Comment:
This is so clean!! :+1: :heart_eyes:

Reply:
What do you think about using decorators to handle the 403 response on error? It would be a separate issue but it might make our code cleaner.

Reply:
Yeah that sounds good to me we can make an issue of it and get around to it

Reply:
#58